File: cpp/easy/2942_find_words_containing_character.cpp
// https://leetcode.com/problems/find-words-containing-character/description/
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> res;
        for(int i = 0; i < words.size(); i++){
            for(auto letter : words[i]){
                if(letter == x){
                    res.push_back(i);
                    break;
                }
            }
        }
        return res;
    }
};

File: python/easy/2942_find_words_containing_character.py
# https://leetcode.com/problems/find-words-containing-character/description/?envType=daily-question&envId=2025-05-24
class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        res = []
        for i in range(len(words)):
            for c in words[i]:
                if c == x:
                    res.append(i)
                    break
        return res
